cfgov/v1/preprocessed/css/molecules/image-text-50-50.less

```diff
@@ -0,0 +1,48 @@
+/* ==========================================================================
+   cfgov-refresh
+   Image and Text 50-50 molecule
+   ========================================================================== */
+
+.m-image-text-50-50 {
+    position: relative;
+    padding-bottom: 60px;
```

Comment:
> Isn't unit the helper mixin?

Haha, right, I just meant more something like `em_unit( @grid_gutter-width )` that called `unit( @grid_gutter-width / @base-font-size-px, em )`. Maybe that's getting too abstracted. It's just that the unit conversion code gets difficult to read in some cases, particularly when division or multiplication is  involved because of the nested parens.